lib/util/DocumentExtractor.js

```diff
@@ -42,9 +42,19 @@ const extractors = ([
     methods: {
       extractFromRequest: request => {
         const documents = [];
-
-        for (let it = 0; it < request.input.body.ids.length; it++) {
-          documents.push({ _id: request.input.body.ids[it] });
+        if (request.input.body && Object.keys(request.input.body).length) {
```

Comment:
This method throws a ReferenceError if there is a non-empty body with no `ids` parameter in it.